src/ipaperftest/core/constants.py

```diff
@@ -109,3 +114,166 @@
 {custom_logs}
 
 """
+
+ANSIBLE_SERVER_CONFIG_PLAYBOOK = """
+---
+- name: Configure server before installation
+  hosts: ipaserver
+  become: yes
+  tasks:
+    - sysctl:
+        name: net.ipv6.conf.all.disable_ipv6
+        value: '0'
+        sysctl_set: yes
+    - replace:
+        path: /etc/hosts
+        regexp: '127.*.*.*\\s*server'
+        replace: '{server_ip} server.{domain} server/'
+"""
+
+ANSIBLE_REPLICA_CONFIG_PLAYBOOK = """
+---
+- name: Configure {replica_name} before installation
+  hosts: {replica_name}
+  become: yes
+  tasks:
+    - lineinfile:
+        path: /etc/hosts
+        line: '{server_ip} server.{domain} server'
+    - lineinfile:
+        path: /etc/resolv.conf
+        line: nameserver {server_ip}
+    - lineinfile:
+        path: /etc/hosts
+        regexp: '127.*.*.*\\s*replica*'
+        state: absent
+    - lineinfile:
+        path: /etc/hosts
+        line: {replica_ip} {replica_name}.{domain} {replica_name}
+"""
+
+ANSIBLE_ENABLE_DATA_COLLECTION_PLAYBOOK = """
+---
+- name: Enable data collection using SAR
+  hosts: ipaserver
+  tasks:
+    - shell:
+        cmd: "nohup sar -o ~/saroutput 2 >/dev/null 2>&1 &"
+"""
+
+ANSIBLE_ENROLLMENTTEST_CLIENT_CONFIG_PLAYBOOK = """
+---
+- name: Configure client machines before installation
+  hosts: ipaclients
+  become: yes
+  tasks:
+    - lineinfile:
+        path: /etc/resolv.conf
+        regexp: ".*"
+        state: absent
+    - lineinfile:
+        path: /etc/resolv.conf
+        line: nameserver {server_ip}
+    - lineinfile:
+        path: /etc/hosts
+        line: {server_ip} server.{domain} server
+"""
+
+ANSIBLE_COUNT_IPA_HOSTS_PLAYBOOK = """
+---
+- name: Count hosts registered in IPA server
+  hosts: ipaserver
+  tasks:
+    - command: "kinit admin"
+      args:
+        stdin: "password"
+    - shell:
+        cmd: "ipa host-find --sizelimit=0 | grep 'Host name:' | wc -l"
+      register: host_find_output
+    - set_fact:
+        host_find_output: "{{{{ host_find_output.stdout }}}}"
+        cacheable: yes
+"""
+
+ANSIBLE_APITEST_CLIENT_CONFIG_PLAYBOOK = """
+---
+- name: Configure client machines before installation
+  hosts: ipaclients
+  become: yes
+  tasks:
+    - lineinfile:
+        path: /etc/resolv.conf
+        regexp: ".*"
+        state: absent
+    - lineinfile:
+        path: /etc/resolv.conf
+        line: nameserver {server_ip}
+    - lineinfile:
+        path: /etc/hosts
+        line: {server_ip} server.{domain} server
+"""
+
+ANSIBLE_AUTHENTICATIONTEST_SERVER_CONFIG_PLAYBOOK = """
+---
+- name: Configure server before execution
+  hosts: ipaserver
+  become: yes
+  tasks:
+    - synchronize:
+        src: "{{{{ item }}}}"
+        dest: "/root"
+        mode: push
+        use_ssh_args: yes
+      with_items:
+        - create-test-data.py
+        - set-password.py
+    - package:
+        name: python3-click
+        state: present
+    - command:
+        cmd: "python create-test-data.py --hosts {amount} --outfile userdata.ldif"
```

Comment:
This needs: --users-per-host {threads}

Otherwise only 10 users per host are going to be created.

Reply:
Fixed, thank you.